engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/WorldPreGenerationScreen.java

```diff
@@ -114,18 +115,7 @@ public void setEnvironment(Context subContext) throws UnresolvedWorldGeneratorEx
     public void initialise() {
 
         final UISlider zoomSlider = find("zoomSlider", UISlider.class);
-        zoomSlider.bindValue(new Binding<Float>() {
-            @Override
-            public Float get() {
-                return zoomValue;
-            }
-
-            @Override
-            public void set(Float value) {
-                zoomValue = value;
-                updatePreview();
-            }
-        });
+        zoomSlider.setUiSliderOnChangeListener(this);
```

Comment:
There's a conflict here although it seems minor. Didn't notice if related but `zoomSlider` here hides a class instance variable. Not necessarily something you caused but could you clarify that if possible?

Reply:
And actually I think I end up getting a related NPE while testing although it could be from a poor conflict fix